build/yamls/flow-visibility/patches/pv/mountPv.yml

```diff
@@ -0,0 +1,14 @@
+- op: add
+  path: /spec/defaults/templates/dataVolumeClaimTemplate
+  value: clickhouse-storage-template
+- op: add
+  path: /spec/templates/volumeClaimTemplates
+  value: 
+    - name: clickhouse-storage-template
+      spec:
+        storageClassName: STORAGECLASS_NAME
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 8Gi
```

Comment:
Actually Kubernetes does not have any enforcement on the storage capacity. The storage size on PV is a required field only for informative. Similarly, the storage size on PVC is required but does not have enforcement on how many space the pod can use. But it will be used to match a PV. Only a PV with a capacity equal to or larger the value specified in the PVC can be bound with this PVC.